src/fetching/deleteEventResponse.ts

```diff
@@ -0,0 +1,17 @@
+import apiUrl from '../utils/apiUrl';
+
+import { ZetkinEventSignup, ZetkinMembership } from '../types/zetkin';
+
+export default async function deleteEventResponse({ eventId, orgId } : ZetkinEventSignup) : Promise<void> {
+    const mUrl = apiUrl('/users/me/memberships');
+    const mRes = await fetch(mUrl);
+    const mData = await mRes.json();
+    const orgMembership = mData.data.find((m : ZetkinMembership) => m.organization.id === orgId);
```

Comment:
This solution is a hack. Maybe leave it like this for now, but add a TODO comment or an issue so we remember to get back to it. It should not be necessary to retrieve the memberships every time. They should be cached.